src/components/body/body.js

```diff
@@ -173,30 +251,44 @@ var Body = {
    * @param  {CANNON.Body} body
    * @param  {CANNON.Shape} shape
    */
-  createWireframe: function (body, shape) {
-    var offset = shape.offset,
-        orientation = shape.orientation,
-        mesh = CANNON.shape2mesh(body).children[0];
-
-    this.wireframe = new THREE.LineSegments(
-      new THREE.EdgesGeometry(mesh.geometry),
-      new THREE.LineBasicMaterial({color: 0xff0000})
-    );
-
-    if (offset) {
-      this.wireframe.offset = offset.clone();
+  createWireframe: function (body) {
+    if (this.wireframe) {
+      this.el.sceneEl.object3D.remove(this.wireframe);
+      delete this.wireframe;
     }
-
-    if (orientation) {
-      orientation.inverse(orientation);
-      this.wireframe.orientation = new THREE.Quaternion(
-        orientation.x,
-        orientation.y,
-        orientation.z,
-        orientation.w
+    this.wireframe = new THREE.Object3D();
+    this.el.sceneEl.object3D.add(this.wireframe);
+
+    var offset, orientation, mesh;
+    for (var i = 0; i < this.body.shapes.length; i++)
+    {
+      offset = this.body.shapeOffsets[i],
+      orientation = this.body.shapeOrientations[i],
+      mesh = CANNON.shape2mesh(this.body).children[i];
+
+      var wireframe = new THREE.LineSegments(
+        new THREE.EdgesGeometry(mesh.geometry),
+        new THREE.LineBasicMaterial({color: 0xff0000})
       );
-    }
 
+      if (offset) {
+        wireframe.position.copy(offset);
+      }
+
+      if (orientation) {
+        orientation.inverse(orientation);
+
+        wireframe.quaternion.set(
+          orientation.x,
+          orientation.y,
+          orientation.z,
+          orientation.w
+        );
```

Comment:
I think `wireframe.quaternion.copy(orientation)` should work here, right? unless there was some hitch between the cannonjs and threejs quaternion...

Reply:
Is that true of going between cannonjs and threejs Vector3's as well?

Reply:
Seems to be the case.